Scripts/Connection/RESTConnector.cs

```diff
@@ -547,42 +542,36 @@ public bool Send(string url, Dictionary<string, string> headers)
                         Headers[kp.Key] = kp.Value;
                 }
 
-                _thread = new Thread(ProcessRequest);
-
-                _thread.Start();
-                return true;
-            }
-
-            private void ProcessRequest()
-            {
                 // This fixes the exception thrown by self-signed certificates.
                 ServicePointManager.ServerCertificateValidationCallback = new RemoteCertificateValidationCallback(delegate { return true; });
 
 #if ENABLE_DEBUGGING
                 Log.Debug("RESTConnector", "DeleteRequest, ProcessRequest {0}", URL);
 #endif
-
-                WebRequest deleteReq = WebRequest.Create(URL);
-
+                UnityWebRequest deleteReq = UnityWebRequest.Delete(URL);
+                deleteReq.method = "DELETE";
                 foreach (var kp in Headers)
-                    deleteReq.Headers.Add(kp.Key, kp.Value);
-                deleteReq.Method = "DELETE";
```

Comment:
As suggestion. We can retrieve all those methods from a constant. (not important in this case since it is internal usage)

Reply:
Yes, actually they have a built in one for UnityWebRequest

https://docs.unity3d.com/ScriptReference/Networking.UnityWebRequest-kHttpVerbDELETE.html

I will change them as I transition to UnityWebRequest.